tests/apis.js

```diff
@@ -0,0 +1,430 @@
+import test from 'ava';
+import uuid from 'uuid';
+import ipsum from 'lorem-ipsum';
+import Promise from 'bluebird';
+import slugify from 'underscore.string/slugify';
+import _ from 'lodash/cloneDeep';
```

Comment:
we need consistency in how we call lodash as well. I would say if it's not all of lodash, then you should title the module according to the method ie: `cloneDeep`. Having `_` mean different things in different files will make maintenance complicated.